pydis_site/apps/api/urls.py

```diff
@@ -33,6 +34,10 @@
     'nominations',
     NominationViewSet
 )
+bot_router.register(
+    'offensive-message',
```

Comment:
I think this should be plural to be consistent with other routes. Docs would need to be updated with the correct route too.